datajunction-server/datajunction_server/api/notifications.py

```diff
@@ -0,0 +1,88 @@
+"""Dependency for notifications"""
+
+import logging
+from typing import Optional
+
+from fastapi import Depends
+from fastapi.responses import JSONResponse
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+
+from datajunction_server.database.history import ActivityType, EntityType, History
+from datajunction_server.database.notification_preference import NotificationPreference
+from datajunction_server.database.user import User
+from datajunction_server.internal.access.authentication.http import SecureAPIRouter
+from datajunction_server.utils import get_and_update_current_user, get_session
+
+router = SecureAPIRouter(tags=["notifications"])
+_logger = logging.getLogger(__name__)
+
+
+def get_notifier():
```

Comment:
This was just moved over from notification.py which I renamed to notification**s**.py to be more consistent with the other router files.